contracts/src/Vault.sol

```diff
@@ -144,31 +138,33 @@ contract Vault is Ownable2Step, IVault {
     ///////////////////////// INFLATION /////////////////////////
 
     /// @notice Try to accrue inflation
-    /// @dev returns the current multiplier
-    function tryInflation() public returns (uint256) {
+    function tryInflation() external only(feeRecipient) {
+        if (block.timestamp < lastKnownTimestamp + 1 days)
+            revert AMKTVaultFeeTooEarly();
         uint256 startingSupply = indexToken.totalSupply();
+        uint256 timestampDiff = block.timestamp - lastKnownTimestamp;
+        uint256 inflation = fmul(startingSupply, timestampDiff * feeScaled);
```

Comment:
I think youre multiplying by the fee? you should divide here

Reply:
this was a miscommunication caused by changed usage of the word `fee`. in this PR, the `fee` is an inflation rate.